### DISCUSSION TITLE
403 error

### DISCUSSION THREAD

**Discussion by johnlopez719** [Avnsx/fansly#136] [Q&A] [answered]
Hey all, I'm getting a 403 error with the downloader. I checked the config file, and the user-agent and authorization token are up to date.
I'm running the latest version of the downloader (0.4.1).

I initially thought the reason was that I just sub to a model and maybe it needs a moment for the downloader to see the subscription. But I then tried models that I have been subbed to for weeks and I am still getting the same error code.
![Screenshot 2023-08-12 164441](https://github.com/Avnsx/fansly-downloader/assets/30753364/a62597fa-2673-4ad6-bb79-8c61786f8195)

**Comment by Avnsx** [comment]
When you upgrade your accounts subscription you need a new auth token, I think.

Remove the auth token in config.ini and re-start fansly downloader so it helps you get a new auth token.

**Reply by johnlopez719** [reply, marked as answer]
[**SOLUTION**]
I had to disable my VPN and  now everything works.
